src/app/core/store/user.interfaces.ts

```diff
@@ -7,4 +7,5 @@ export interface UserResponse {
   firstName: string;
   lastName: string;
   email: string;
+  role: string;
```

Comment:
Może lepiej wydzielmy UserRole jako osobny typ: 
`export type UserRole = 'user' | 'admin';`
a następnie ustalmy `role` jako `UserRole`, zamiast `string`.
W ten sposób zawięzimy typ, bo innych roli nie mamy. Jednocześnie pozwoli nam to w przyszłości na dostęp do obiektów specyficznych dla usera i admina za pomocą nawiasów kwadratowych np: 
`const menu = {
user: [...items],
admin: [...items]
};`
`const role: UserRole = this.store.select(loggedUserRole);`
`console.log(menu[role]);` 
Wyświetli menu usera lub admina w zależności od aktualnej roli bez żadnych błędów 😉